[2019-06-12 06:51:38] request.INFO: Matched route "admin_module_manage_action". {"route":"admin_module_manage_action","route_parameters":{"_controller":"PrestaShopBundle\\Controller\\Admin\\Improve\\ModuleController::moduleAction","action":"upgrade","module_name":"ps_facetedsearch","_route":"admin_module_manage_action"},"request_uri":"http://cadouri-new.test/admin038sx7bbg/index.php/improve/modules/manage/action/upgrade/ps_facetedsearch?_token=h1Ew9D2ovCX3XyJUgxXT-DSM8a7-n8_MmgXaf9KwJrU","method":"POST"} []
[2019-06-12 06:51:38] security.DEBUG: Read existing security token from the session. {"key":"_security_main","token_class":"Symfony\\Component\\Security\\Core\\Authentication\\Token\\UsernamePasswordToken"} []
[2019-06-12 06:51:38] security.DEBUG: User was reloaded from a user provider. {"provider":"PrestaShopBundle\\Security\\Admin\\EmployeeProvider","username":"alex-tech@fruitware.ru"} []
[2019-06-12 06:51:43] app.ERROR: Exception thrown by module ps_facetedsearch on upgrade. Error sent by Addons. You may need to be logged. [] []
[2019-06-12 06:51:43] security.DEBUG: Stored the security token in the session. {"key":"_security_main"} []
[2019-06-12 10:15:29] request.INFO: Matched route "admin_module_manage_action". {"route":"admin_module_manage_action","route_parameters":{"_controller":"PrestaShopBundle\\Controller\\Admin\\Improve\\ModuleController::moduleAction","action":"upgrade","module_name":"ps_facetedsearch","_route":"admin_module_manage_action"},"request_uri":"http://cadouri-new.test/admin038sx7bbg/index.php/improve/modules/manage/action/upgrade/ps_facetedsearch?_token=UKSeYKKAmFl_bgxm2DrtFarq2TkV6DhYuNOZ7cNuvgs","method":"POST"} []
[2019-06-12 10:15:29] security.DEBUG: Read existing security token from the session. {"key":"_security_main","token_class":"Symfony\\Component\\Security\\Core\\Authentication\\Token\\UsernamePasswordToken"} []
[2019-06-12 10:15:29] security.DEBUG: User was reloaded from a user provider. {"provider":"PrestaShopBundle\\Security\\Admin\\EmployeeProvider","username":"alex-tech@fruitware.ru"} []
[2019-06-12 10:15:34] app.ERROR: Exception thrown by module ps_facetedsearch on upgrade. Error sent by Addons. You may need to be logged. [] []
[2019-06-12 10:15:34] security.DEBUG: Stored the security token in the session. {"key":"_security_main"} []
[2019-06-13 21:59:11] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Filesystem\Exception\IOException: "Failed to remove file "C:\Programs\XAMPP\htdocs/var/cache/prod\translations\catalogue..NXhscRe.php": unlink(C:\Programs\XAMPP\htdocs/var/cache/prod\translations\catalogue..NXhscRe.php): Permission denied." at C:\Programs\XAMPP\htdocs\vendor\symfony\symfony\src\Symfony\Component\Filesystem\Filesystem.php line 183 {"exception":"[object] (Symfony\\Component\\Filesystem\\Exception\\IOException(code: 0): Failed to remove file \"C:\\Programs\\XAMPP\\htdocs/var/cache/prod\\translations\\catalogue..NXhscRe.php\": unlink(C:\\Programs\\XAMPP\\htdocs/var/cache/prod\\translations\\catalogue..NXhscRe.php): Permission denied. at C:\\Programs\\XAMPP\\htdocs\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\Filesystem\\Filesystem.php:183)"} []
[2019-06-13 21:59:11] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Filesystem\Exception\IOException: "Cannot rename "C:\Programs\XAMPP\htdocs\var\cache\prod\translations\cat3CF5.tmp" to "C:\Programs\XAMPP\htdocs/var/cache/prod\translations/catalogue..NXhscRe.php"." at C:\Programs\XAMPP\htdocs\vendor\symfony\symfony\src\Symfony\Component\Filesystem\Filesystem.php line 289 {"exception":"[object] (Symfony\\Component\\Filesystem\\Exception\\IOException(code: 0): Cannot rename \"C:\\Programs\\XAMPP\\htdocs\\var\\cache\\prod\\translations\\cat3CF5.tmp\" to \"C:\\Programs\\XAMPP\\htdocs/var/cache/prod\\translations/catalogue..NXhscRe.php\". at C:\\Programs\\XAMPP\\htdocs\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\Filesystem\\Filesystem.php:289)"} []
[2019-06-13 21:59:11] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Filesystem\Exception\IOException: "Cannot rename "C:\Programs\XAMPP\htdocs\var\cache\prod\translations\cat3D19.tmp" to "C:\Programs\XAMPP\htdocs/var/cache/prod\translations/catalogue..NXhscRe.php.meta"." at C:\Programs\XAMPP\htdocs\vendor\symfony\symfony\src\Symfony\Component\Filesystem\Filesystem.php line 289 {"exception":"[object] (Symfony\\Component\\Filesystem\\Exception\\IOException(code: 0): Cannot rename \"C:\\Programs\\XAMPP\\htdocs\\var\\cache\\prod\\translations\\cat3D19.tmp\" to \"C:\\Programs\\XAMPP\\htdocs/var/cache/prod\\translations/catalogue..NXhscRe.php.meta\". at C:\\Programs\\XAMPP\\htdocs\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\Filesystem\\Filesystem.php:289)"} []
[2019-06-13 21:59:11] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Filesystem\Exception\IOException: "Cannot rename "C:\Programs\XAMPP\htdocs\var\cache\prod\translations\cat3DEC.tmp" to "C:\Programs\XAMPP\htdocs/var/cache/prod\translations/catalogue..NXhscRe.php.meta"." at C:\Programs\XAMPP\htdocs\vendor\symfony\symfony\src\Symfony\Component\Filesystem\Filesystem.php line 289 {"exception":"[object] (Symfony\\Component\\Filesystem\\Exception\\IOException(code: 0): Cannot rename \"C:\\Programs\\XAMPP\\htdocs\\var\\cache\\prod\\translations\\cat3DEC.tmp\" to \"C:\\Programs\\XAMPP\\htdocs/var/cache/prod\\translations/catalogue..NXhscRe.php.meta\". at C:\\Programs\\XAMPP\\htdocs\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\Filesystem\\Filesystem.php:289)"} []
[2019-06-13 21:59:12] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Filesystem\Exception\IOException: "Cannot rename "C:\Programs\XAMPP\htdocs\var\cache\prod\translations\cat4154.tmp" to "C:\Programs\XAMPP\htdocs/var/cache/prod\translations/catalogue..NXhscRe.php"." at C:\Programs\XAMPP\htdocs\vendor\symfony\symfony\src\Symfony\Component\Filesystem\Filesystem.php line 289 {"exception":"[object] (Symfony\\Component\\Filesystem\\Exception\\IOException(code: 0): Cannot rename \"C:\\Programs\\XAMPP\\htdocs\\var\\cache\\prod\\translations\\cat4154.tmp\" to \"C:\\Programs\\XAMPP\\htdocs/var/cache/prod\\translations/catalogue..NXhscRe.php\". at C:\\Programs\\XAMPP\\htdocs\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\Filesystem\\Filesystem.php:289)"} []
[2019-06-13 21:59:12] cache.WARNING: Failed to save key "%5BSymfony%5CBundle%5CTwigBundle%5CController%5CExceptionController%23showAction%5D%5B1%5D" (array) {"key":"%5BSymfony%5CBundle%5CTwigBundle%5CController%5CExceptionController%23showAction%5D%5B1%5D","type":"array","exception":"[object] (ErrorException(code: 0): rename(C:\\Programs\\XAMPP\\htdocs\\var\\cache\\prod\\pools\\926Sk+OIma\\5d029d00b6a828.32435117,C:\\Programs\\XAMPP\\htdocs\\var\\cache\\prod\\pools\\926Sk+OIma\\-\\J\\YU-hJZt1CGILbBXWF3bl): Access is denied. (code: 5) at C:\\Programs\\XAMPP\\htdocs\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\Cache\\Traits\\FilesystemCommonTrait.php:93)"} []
[2019-06-13 21:59:13] request.CRITICAL: Exception thrown when handling an exception (RuntimeException: Failed to write cache file "C:\Programs\XAMPP\htdocs\var\cache\prod/twig/66/663cf1073da274f4c245c0bfcfd7649fb99df5d15ab45428eb81267377a9b89c.php". at C:\Programs\XAMPP\htdocs\vendor\twig\twig\lib\Twig\Cache\Filesystem.php line 80) {"exception":"[object] (RuntimeException(code: 0): Failed to write cache file \"C:\\Programs\\XAMPP\\htdocs\\var\\cache\\prod/twig/66/663cf1073da274f4c245c0bfcfd7649fb99df5d15ab45428eb81267377a9b89c.php\". at C:\\Programs\\XAMPP\\htdocs\\vendor\\twig\\twig\\lib\\Twig\\Cache\\Filesystem.php:80)"} []
[2019-06-13 21:59:13] request.CRITICAL: Exception thrown when handling an exception (RuntimeException: Failed to write cache file "C:\Programs\XAMPP\htdocs\var\cache\prod/twig/66/663cf1073da274f4c245c0bfcfd7649fb99df5d15ab45428eb81267377a9b89c.php". at C:\Programs\XAMPP\htdocs\vendor\twig\twig\lib\Twig\Cache\Filesystem.php line 80) {"exception":"[object] (RuntimeException(code: 0): Failed to write cache file \"C:\\Programs\\XAMPP\\htdocs\\var\\cache\\prod/twig/66/663cf1073da274f4c245c0bfcfd7649fb99df5d15ab45428eb81267377a9b89c.php\". at C:\\Programs\\XAMPP\\htdocs\\vendor\\twig\\twig\\lib\\Twig\\Cache\\Filesystem.php:80)"} []
[2019-06-13 21:59:34] app.ERROR: Data from PrestaShop Addons is invalid, and cannot fallback on cache.  {"exception":"cURL error 28: Operation timed out after 5000 milliseconds with 137931 bytes received"} []
[2019-06-13 22:02:48] request.INFO: Matched route "admin_performance". {"route":"admin_performance","route_parameters":{"_controller":"PrestaShopBundle\\Controller\\Admin\\Configure\\AdvancedParameters\\PerformanceController::indexAction","_legacy_controller":"AdminPerformance","_legacy_link":"AdminPerformance","_route":"admin_performance"},"request_uri":"http://localhost/admin038sx7bbg/index.php/configure/advanced/performance/?_token=L0-y-Zlbpt36jGHPzfCt0PPyeV4i-497HHnpPz-cDHc","method":"GET"} []
[2019-06-13 22:02:48] security.DEBUG: Read existing security token from the session. {"key":"_security_main","token_class":"Symfony\\Component\\Security\\Core\\Authentication\\Token\\UsernamePasswordToken"} []
[2019-06-13 22:02:48] security.DEBUG: User was reloaded from a user provider. {"provider":"PrestaShopBundle\\Security\\Admin\\EmployeeProvider","username":"alex-tech@fruitware.ru"} []
[2019-06-13 22:03:02] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\FatalErrorException: "Error: Maximum execution time of 30 seconds exceeded" at C:\Programs\XAMPP\htdocs\classes\Tools.php line 2100 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalErrorException(code: 0): Error: Maximum execution time of 30 seconds exceeded at C:\\Programs\\XAMPP\\htdocs\\classes\\Tools.php:2100)"} []
[2019-06-13 22:03:02] security.DEBUG: Stored the security token in the session. {"key":"_security_main"} []
[2019-06-13 22:04:30] request.INFO: Matched route "admin_performance". {"route":"admin_performance","route_parameters":{"_controller":"PrestaShopBundle\\Controller\\Admin\\Configure\\AdvancedParameters\\PerformanceController::indexAction","_legacy_controller":"AdminPerformance","_legacy_link":"AdminPerformance","_route":"admin_performance"},"request_uri":"http://localhost/admin038sx7bbg/index.php/configure/advanced/performance/?_token=L0-y-Zlbpt36jGHPzfCt0PPyeV4i-497HHnpPz-cDHc","method":"GET"} []
[2019-06-13 22:04:30] security.DEBUG: Read existing security token from the session. {"key":"_security_main","token_class":"Symfony\\Component\\Security\\Core\\Authentication\\Token\\UsernamePasswordToken"} []
[2019-06-13 22:04:30] security.DEBUG: User was reloaded from a user provider. {"provider":"PrestaShopBundle\\Security\\Admin\\EmployeeProvider","username":"alex-tech@fruitware.ru"} []
[2019-06-13 22:04:56] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\FatalErrorException: "Error: Maximum execution time of 30 seconds exceeded" at C:\Programs\XAMPP\htdocs\classes\Tools.php line 2100 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalErrorException(code: 0): Error: Maximum execution time of 30 seconds exceeded at C:\\Programs\\XAMPP\\htdocs\\classes\\Tools.php:2100)"} []
[2019-06-13 22:04:56] security.DEBUG: Stored the security token in the session. {"key":"_security_main"} []
[2019-06-13 22:08:01] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Filesystem\Exception\IOException: "Cannot rename "C:\Programs\XAMPP\htdocs\var\cache\prod\translations\cat6867.tmp" to "C:\Programs\XAMPP\htdocs/var/cache/prod\translations/catalogue..NXhscRe.php.meta"." at C:\Programs\XAMPP\htdocs\vendor\symfony\symfony\src\Symfony\Component\Filesystem\Filesystem.php line 289 {"exception":"[object] (Symfony\\Component\\Filesystem\\Exception\\IOException(code: 0): Cannot rename \"C:\\Programs\\XAMPP\\htdocs\\var\\cache\\prod\\translations\\cat6867.tmp\" to \"C:\\Programs\\XAMPP\\htdocs/var/cache/prod\\translations/catalogue..NXhscRe.php.meta\". at C:\\Programs\\XAMPP\\htdocs\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\Filesystem\\Filesystem.php:289)"} []
[2019-06-13 22:08:01] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Filesystem\Exception\IOException: "Cannot rename "C:\Programs\XAMPP\htdocs\var\cache\prod\translations\cat68BC.tmp" to "C:\Programs\XAMPP\htdocs/var/cache/prod\translations/catalogue..NXhscRe.php"." at C:\Programs\XAMPP\htdocs\vendor\symfony\symfony\src\Symfony\Component\Filesystem\Filesystem.php line 289 {"exception":"[object] (Symfony\\Component\\Filesystem\\Exception\\IOException(code: 0): Cannot rename \"C:\\Programs\\XAMPP\\htdocs\\var\\cache\\prod\\translations\\cat68BC.tmp\" to \"C:\\Programs\\XAMPP\\htdocs/var/cache/prod\\translations/catalogue..NXhscRe.php\". at C:\\Programs\\XAMPP\\htdocs\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\Filesystem\\Filesystem.php:289)"} []
[2019-06-13 22:08:01] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Filesystem\Exception\IOException: "Cannot rename "C:\Programs\XAMPP\htdocs\var\cache\prod\translations\cat6945.tmp" to "C:\Programs\XAMPP\htdocs/var/cache/prod\translations/catalogue..NXhscRe.php"." at C:\Programs\XAMPP\htdocs\vendor\symfony\symfony\src\Symfony\Component\Filesystem\Filesystem.php line 289 {"exception":"[object] (Symfony\\Component\\Filesystem\\Exception\\IOException(code: 0): Cannot rename \"C:\\Programs\\XAMPP\\htdocs\\var\\cache\\prod\\translations\\cat6945.tmp\" to \"C:\\Programs\\XAMPP\\htdocs/var/cache/prod\\translations/catalogue..NXhscRe.php\". at C:\\Programs\\XAMPP\\htdocs\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\Filesystem\\Filesystem.php:289)"} []
[2019-06-13 22:08:01] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Filesystem\Exception\IOException: "Cannot rename "C:\Programs\XAMPP\htdocs\var\cache\prod\translations\cat6948.tmp" to "C:\Programs\XAMPP\htdocs/var/cache/prod\translations/catalogue..NXhscRe.php.meta"." at C:\Programs\XAMPP\htdocs\vendor\symfony\symfony\src\Symfony\Component\Filesystem\Filesystem.php line 289 {"exception":"[object] (Symfony\\Component\\Filesystem\\Exception\\IOException(code: 0): Cannot rename \"C:\\Programs\\XAMPP\\htdocs\\var\\cache\\prod\\translations\\cat6948.tmp\" to \"C:\\Programs\\XAMPP\\htdocs/var/cache/prod\\translations/catalogue..NXhscRe.php.meta\". at C:\\Programs\\XAMPP\\htdocs\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\Filesystem\\Filesystem.php:289)"} []
[2019-06-13 22:08:31] request.INFO: Matched route "admin_performance". {"route":"admin_performance","route_parameters":{"_controller":"PrestaShopBundle\\Controller\\Admin\\Configure\\AdvancedParameters\\PerformanceController::indexAction","_legacy_controller":"AdminPerformance","_legacy_link":"AdminPerformance","_route":"admin_performance"},"request_uri":"http://localhost/admin038sx7bbg/index.php/configure/advanced/performance/?_token=L0-y-Zlbpt36jGHPzfCt0PPyeV4i-497HHnpPz-cDHc","method":"GET"} []
[2019-06-13 22:08:31] security.DEBUG: Read existing security token from the session. {"key":"_security_main","token_class":"Symfony\\Component\\Security\\Core\\Authentication\\Token\\UsernamePasswordToken"} []
[2019-06-13 22:08:31] security.DEBUG: User was reloaded from a user provider. {"provider":"PrestaShopBundle\\Security\\Admin\\EmployeeProvider","username":"alex-tech@fruitware.ru"} []
[2019-06-13 22:08:55] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\FatalErrorException: "Error: Maximum execution time of 30 seconds exceeded" at C:\Programs\XAMPP\htdocs\classes\Tools.php line 2100 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalErrorException(code: 0): Error: Maximum execution time of 30 seconds exceeded at C:\\Programs\\XAMPP\\htdocs\\classes\\Tools.php:2100)"} []
[2019-06-13 22:08:55] security.DEBUG: Stored the security token in the session. {"key":"_security_main"} []
[2019-06-13 22:12:55] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\FatalErrorException: "Error: Maximum execution time of 30 seconds exceeded" at C:\Programs\XAMPP\htdocs\vendor\symfony\symfony\src\Symfony\Component\Finder\Iterator\MultiplePcreFilterIterator.php line 65 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalErrorException(code: 0): Error: Maximum execution time of 30 seconds exceeded at C:\\Programs\\XAMPP\\htdocs\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\Finder\\Iterator\\MultiplePcreFilterIterator.php:65)"} []
[2019-06-14 14:31:51] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Filesystem\Exception\IOException: "Cannot rename "C:\Programs\XAMPP\htdocs\var\cache\prod\translations\catEBEF.tmp" to "C:\Programs\XAMPP\htdocs/var/cache/prod\translations/catalogue..NXhscRe.php"." at C:\Programs\XAMPP\htdocs\vendor\symfony\symfony\src\Symfony\Component\Filesystem\Filesystem.php line 289 {"exception":"[object] (Symfony\\Component\\Filesystem\\Exception\\IOException(code: 0): Cannot rename \"C:\\Programs\\XAMPP\\htdocs\\var\\cache\\prod\\translations\\catEBEF.tmp\" to \"C:\\Programs\\XAMPP\\htdocs/var/cache/prod\\translations/catalogue..NXhscRe.php\". at C:\\Programs\\XAMPP\\htdocs\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\Filesystem\\Filesystem.php:289)"} []
[2019-06-14 14:32:05] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Filesystem\Exception\IOException: "Cannot rename "C:\Programs\XAMPP\htdocs\var\cache\prod\translations\cat24B4.tmp" to "C:\Programs\XAMPP\htdocs/var/cache/prod\translations/catalogue..NXhscRe.php.meta"." at C:\Programs\XAMPP\htdocs\vendor\symfony\symfony\src\Symfony\Component\Filesystem\Filesystem.php line 289 {"exception":"[object] (Symfony\\Component\\Filesystem\\Exception\\IOException(code: 0): Cannot rename \"C:\\Programs\\XAMPP\\htdocs\\var\\cache\\prod\\translations\\cat24B4.tmp\" to \"C:\\Programs\\XAMPP\\htdocs/var/cache/prod\\translations/catalogue..NXhscRe.php.meta\". at C:\\Programs\\XAMPP\\htdocs\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\Filesystem\\Filesystem.php:289)"} []
[2019-06-14 14:32:05] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Filesystem\Exception\IOException: "Cannot rename "C:\Programs\XAMPP\htdocs\var\cache\prod\translations\cat251D.tmp" to "C:\Programs\XAMPP\htdocs/var/cache/prod\translations/catalogue..NXhscRe.php.meta"." at C:\Programs\XAMPP\htdocs\vendor\symfony\symfony\src\Symfony\Component\Filesystem\Filesystem.php line 289 {"exception":"[object] (Symfony\\Component\\Filesystem\\Exception\\IOException(code: 0): Cannot rename \"C:\\Programs\\XAMPP\\htdocs\\var\\cache\\prod\\translations\\cat251D.tmp\" to \"C:\\Programs\\XAMPP\\htdocs/var/cache/prod\\translations/catalogue..NXhscRe.php.meta\". at C:\\Programs\\XAMPP\\htdocs\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\Filesystem\\Filesystem.php:289)"} []
[2019-06-14 14:32:05] request.CRITICAL: Uncaught PHP Exception SmartyException: "Invalid compiled template for 'controllers/modules/login_addons.tpl'" at C:\Programs\XAMPP\htdocs\vendor\smarty\smarty\libs\sysplugins\smarty_template_resource_base.php line 116 {"exception":"[object] (SmartyException(code: 0): Invalid compiled template for 'controllers/modules/login_addons.tpl' at C:\\Programs\\XAMPP\\htdocs\\vendor\\smarty\\smarty\\libs\\sysplugins\\smarty_template_resource_base.php:116)"} []
[2019-06-14 14:32:05] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Filesystem\Exception\IOException: "Cannot rename "C:\Programs\XAMPP\htdocs\var\cache\prod\translations\cat26AE.tmp" to "C:\Programs\XAMPP\htdocs/var/cache/prod\translations/catalogue..NXhscRe.php.meta"." at C:\Programs\XAMPP\htdocs\vendor\symfony\symfony\src\Symfony\Component\Filesystem\Filesystem.php line 289 {"exception":"[object] (Symfony\\Component\\Filesystem\\Exception\\IOException(code: 0): Cannot rename \"C:\\Programs\\XAMPP\\htdocs\\var\\cache\\prod\\translations\\cat26AE.tmp\" to \"C:\\Programs\\XAMPP\\htdocs/var/cache/prod\\translations/catalogue..NXhscRe.php.meta\". at C:\\Programs\\XAMPP\\htdocs\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\Filesystem\\Filesystem.php:289)"} []
[2019-06-14 14:32:06] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Filesystem\Exception\IOException: "Cannot rename "C:\Programs\XAMPP\htdocs\var\cache\prod\translations\cat283E.tmp" to "C:\Programs\XAMPP\htdocs/var/cache/prod\translations/catalogue..NXhscRe.php"." at C:\Programs\XAMPP\htdocs\vendor\symfony\symfony\src\Symfony\Component\Filesystem\Filesystem.php line 289 {"exception":"[object] (Symfony\\Component\\Filesystem\\Exception\\IOException(code: 0): Cannot rename \"C:\\Programs\\XAMPP\\htdocs\\var\\cache\\prod\\translations\\cat283E.tmp\" to \"C:\\Programs\\XAMPP\\htdocs/var/cache/prod\\translations/catalogue..NXhscRe.php\". at C:\\Programs\\XAMPP\\htdocs\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\Filesystem\\Filesystem.php:289)"} []
[2019-06-14 14:50:29] request.INFO: Matched route "admin_module_catalog". {"route":"admin_module_catalog","route_parameters":{"_controller":"PrestaShopBundle\\Controller\\Admin\\Improve\\ModuleController::catalogAction","_legacy_controller":"AdminModulesCatalog","_legacy_link":"AdminModulesCatalog","_route":"admin_module_catalog"},"request_uri":"http://localhost/admin038sx7bbg/index.php/improve/modules/catalog?_token=MsjfUc6SKm1_YgR0zm2zx6KltpOwEojyMSGBvh2uLoE","method":"GET"} []
[2019-06-14 14:50:32] security.DEBUG: Read existing security token from the session. {"key":"_security_main","token_class":"Symfony\\Component\\Security\\Core\\Authentication\\Token\\UsernamePasswordToken"} []
[2019-06-14 14:50:32] security.DEBUG: User was reloaded from a user provider. {"provider":"PrestaShopBundle\\Security\\Admin\\EmployeeProvider","username":"alex-tech@fruitware.ru"} []
[2019-06-14 14:50:33] request.CRITICAL: Uncaught PHP Exception Twig_Error_Runtime: "An exception has been thrown during the rendering of a template ("Cannot rename "C:\Programs\XAMPP\htdocs\var\cache\prod\translations\cat25E3.tmp" to "C:\Programs\XAMPP\htdocs/var/cache/prod\translations/catalogue..NXhscRe.php".")." at C:\Programs\XAMPP\htdocs\src\PrestaShopBundle\Resources\views\Admin\layout.html.twig line 34 {"exception":"[object] (Twig_Error_Runtime(code: 0): An exception has been thrown during the rendering of a template (\"Cannot rename \"C:\\Programs\\XAMPP\\htdocs\\var\\cache\\prod\\translations\\cat25E3.tmp\" to \"C:\\Programs\\XAMPP\\htdocs/var/cache/prod\\translations/catalogue..NXhscRe.php\".\"). at C:\\Programs\\XAMPP\\htdocs\\src\\PrestaShopBundle\\Resources\\views\\Admin\\layout.html.twig:34, Symfony\\Component\\Filesystem\\Exception\\IOException(code: 0): Cannot rename \"C:\\Programs\\XAMPP\\htdocs\\var\\cache\\prod\\translations\\cat25E3.tmp\" to \"C:\\Programs\\XAMPP\\htdocs/var/cache/prod\\translations/catalogue..NXhscRe.php\". at C:\\Programs\\XAMPP\\htdocs\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\Filesystem\\Filesystem.php:289)"} []
[2019-06-14 14:50:33] security.DEBUG: Stored the security token in the session. {"key":"_security_main"} []
[2019-06-14 14:50:53] request.CRITICAL: Uncaught PHP Exception SmartyException: "Invalid compiled template for 'controllers/modules/login_addons.tpl'" at C:\Programs\XAMPP\htdocs\vendor\smarty\smarty\libs\sysplugins\smarty_template_resource_base.php line 116 {"exception":"[object] (SmartyException(code: 0): Invalid compiled template for 'controllers/modules/login_addons.tpl' at C:\\Programs\\XAMPP\\htdocs\\vendor\\smarty\\smarty\\libs\\sysplugins\\smarty_template_resource_base.php:116)"} []
[2019-06-14 14:50:53] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Filesystem\Exception\IOException: "Cannot rename "C:\Programs\XAMPP\htdocs\var\cache\prod\translations\cat773A.tmp" to "C:\Programs\XAMPP\htdocs/var/cache/prod\translations/catalogue..NXhscRe.php"." at C:\Programs\XAMPP\htdocs\vendor\symfony\symfony\src\Symfony\Component\Filesystem\Filesystem.php line 289 {"exception":"[object] (Symfony\\Component\\Filesystem\\Exception\\IOException(code: 0): Cannot rename \"C:\\Programs\\XAMPP\\htdocs\\var\\cache\\prod\\translations\\cat773A.tmp\" to \"C:\\Programs\\XAMPP\\htdocs/var/cache/prod\\translations/catalogue..NXhscRe.php\". at C:\\Programs\\XAMPP\\htdocs\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\Filesystem\\Filesystem.php:289)"} []
[2019-06-14 14:50:54] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Filesystem\Exception\IOException: "Cannot rename "C:\Programs\XAMPP\htdocs\var\cache\prod\translations\cat77FB.tmp" to "C:\Programs\XAMPP\htdocs/var/cache/prod\translations/catalogue..NXhscRe.php.meta"." at C:\Programs\XAMPP\htdocs\vendor\symfony\symfony\src\Symfony\Component\Filesystem\Filesystem.php line 289 {"exception":"[object] (Symfony\\Component\\Filesystem\\Exception\\IOException(code: 0): Cannot rename \"C:\\Programs\\XAMPP\\htdocs\\var\\cache\\prod\\translations\\cat77FB.tmp\" to \"C:\\Programs\\XAMPP\\htdocs/var/cache/prod\\translations/catalogue..NXhscRe.php.meta\". at C:\\Programs\\XAMPP\\htdocs\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\Filesystem\\Filesystem.php:289)"} []
[2019-06-14 14:50:54] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Filesystem\Exception\IOException: "Cannot rename "C:\Programs\XAMPP\htdocs\var\cache\prod\translations\cat7A08.tmp" to "C:\Programs\XAMPP\htdocs/var/cache/prod\translations/catalogue..NXhscRe.php.meta"." at C:\Programs\XAMPP\htdocs\vendor\symfony\symfony\src\Symfony\Component\Filesystem\Filesystem.php line 289 {"exception":"[object] (Symfony\\Component\\Filesystem\\Exception\\IOException(code: 0): Cannot rename \"C:\\Programs\\XAMPP\\htdocs\\var\\cache\\prod\\translations\\cat7A08.tmp\" to \"C:\\Programs\\XAMPP\\htdocs/var/cache/prod\\translations/catalogue..NXhscRe.php.meta\". at C:\\Programs\\XAMPP\\htdocs\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\Filesystem\\Filesystem.php:289)"} []
[2019-06-14 14:57:58] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Filesystem\Exception\IOException: "Cannot rename "C:\Programs\XAMPP\htdocs\var\cache\prod\translations\catFD56.tmp" to "C:\Programs\XAMPP\htdocs/var/cache/prod\translations/catalogue..NXhscRe.php"." at C:\Programs\XAMPP\htdocs\vendor\symfony\symfony\src\Symfony\Component\Filesystem\Filesystem.php line 289 {"exception":"[object] (Symfony\\Component\\Filesystem\\Exception\\IOException(code: 0): Cannot rename \"C:\\Programs\\XAMPP\\htdocs\\var\\cache\\prod\\translations\\catFD56.tmp\" to \"C:\\Programs\\XAMPP\\htdocs/var/cache/prod\\translations/catalogue..NXhscRe.php\". at C:\\Programs\\XAMPP\\htdocs\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\Filesystem\\Filesystem.php:289)"} []
[2019-06-14 14:57:59] request.CRITICAL: Uncaught PHP Exception SmartyException: "unable to write file C:\Programs\XAMPP\htdocs\var\cache\prod\smarty\compile\82\b8\b6\82b8b64824cbd540ac132798d4432bdc19cefa49_0.file.login_addons.tpl.php" at C:\Programs\XAMPP\htdocs\vendor\smarty\smarty\libs\sysplugins\smarty_internal_runtime_writefile.php line 90 {"exception":"[object] (SmartyException(code: 0): unable to write file C:\\Programs\\XAMPP\\htdocs\\var\\cache\\prod\\smarty\\compile\\82\\b8\\b6\\82b8b64824cbd540ac132798d4432bdc19cefa49_0.file.login_addons.tpl.php at C:\\Programs\\XAMPP\\htdocs\\vendor\\smarty\\smarty\\libs\\sysplugins\\smarty_internal_runtime_writefile.php:90)"} []
[2019-06-14 14:57:59] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Filesystem\Exception\IOException: "Cannot rename "C:\Programs\XAMPP\htdocs\var\cache\prod\translations\cat151.tmp" to "C:\Programs\XAMPP\htdocs/var/cache/prod\translations/catalogue..NXhscRe.php"." at C:\Programs\XAMPP\htdocs\vendor\symfony\symfony\src\Symfony\Component\Filesystem\Filesystem.php line 289 {"exception":"[object] (Symfony\\Component\\Filesystem\\Exception\\IOException(code: 0): Cannot rename \"C:\\Programs\\XAMPP\\htdocs\\var\\cache\\prod\\translations\\cat151.tmp\" to \"C:\\Programs\\XAMPP\\htdocs/var/cache/prod\\translations/catalogue..NXhscRe.php\". at C:\\Programs\\XAMPP\\htdocs\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\Filesystem\\Filesystem.php:289)"} []
[2019-06-14 22:38:20] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Filesystem\Exception\IOException: "Cannot rename "C:\Programs\XAMPP\htdocs\var\cache\prod\translations\catE908.tmp" to "C:\Programs\XAMPP\htdocs/var/cache/prod\translations/catalogue..NXhscRe.php"." at C:\Programs\XAMPP\htdocs\vendor\symfony\symfony\src\Symfony\Component\Filesystem\Filesystem.php line 289 {"exception":"[object] (Symfony\\Component\\Filesystem\\Exception\\IOException(code: 0): Cannot rename \"C:\\Programs\\XAMPP\\htdocs\\var\\cache\\prod\\translations\\catE908.tmp\" to \"C:\\Programs\\XAMPP\\htdocs/var/cache/prod\\translations/catalogue..NXhscRe.php\". at C:\\Programs\\XAMPP\\htdocs\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\Filesystem\\Filesystem.php:289)"} []
[2019-06-14 22:38:29] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Filesystem\Exception\IOException: "Cannot rename "C:\Programs\XAMPP\htdocs\var\cache\prod\translations\catCD7.tmp" to "C:\Programs\XAMPP\htdocs/var/cache/prod\translations/catalogue..NXhscRe.php"." at C:\Programs\XAMPP\htdocs\vendor\symfony\symfony\src\Symfony\Component\Filesystem\Filesystem.php line 289 {"exception":"[object] (Symfony\\Component\\Filesystem\\Exception\\IOException(code: 0): Cannot rename \"C:\\Programs\\XAMPP\\htdocs\\var\\cache\\prod\\translations\\catCD7.tmp\" to \"C:\\Programs\\XAMPP\\htdocs/var/cache/prod\\translations/catalogue..NXhscRe.php\". at C:\\Programs\\XAMPP\\htdocs\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\Filesystem\\Filesystem.php:289)"} []
[2019-06-14 22:38:36] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Filesystem\Exception\IOException: "Cannot rename "C:\Programs\XAMPP\htdocs\var\cache\prod\translations\cat2ACF.tmp" to "C:\Programs\XAMPP\htdocs/var/cache/prod\translations/catalogue..NXhscRe.php"." at C:\Programs\XAMPP\htdocs\vendor\symfony\symfony\src\Symfony\Component\Filesystem\Filesystem.php line 289 {"exception":"[object] (Symfony\\Component\\Filesystem\\Exception\\IOException(code: 0): Cannot rename \"C:\\Programs\\XAMPP\\htdocs\\var\\cache\\prod\\translations\\cat2ACF.tmp\" to \"C:\\Programs\\XAMPP\\htdocs/var/cache/prod\\translations/catalogue..NXhscRe.php\". at C:\\Programs\\XAMPP\\htdocs\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\Filesystem\\Filesystem.php:289)"} []
[2019-06-14 22:39:41] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Filesystem\Exception\IOException: "Cannot rename "C:\Programs\XAMPP\htdocs\var\cache\prod\translations\cat2AE2.tmp" to "C:\Programs\XAMPP\htdocs/var/cache/prod\translations/catalogue..NXhscRe.php.meta"." at C:\Programs\XAMPP\htdocs\vendor\symfony\symfony\src\Symfony\Component\Filesystem\Filesystem.php line 289 {"exception":"[object] (Symfony\\Component\\Filesystem\\Exception\\IOException(code: 0): Cannot rename \"C:\\Programs\\XAMPP\\htdocs\\var\\cache\\prod\\translations\\cat2AE2.tmp\" to \"C:\\Programs\\XAMPP\\htdocs/var/cache/prod\\translations/catalogue..NXhscRe.php.meta\". at C:\\Programs\\XAMPP\\htdocs\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\Filesystem\\Filesystem.php:289)"} []
[2019-06-14 22:40:13] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Filesystem\Exception\IOException: "Cannot rename "C:\Programs\XAMPP\htdocs\var\cache\prod\translations\catA970.tmp" to "C:\Programs\XAMPP\htdocs/var/cache/prod\translations/catalogue..NXhscRe.php"." at C:\Programs\XAMPP\htdocs\vendor\symfony\symfony\src\Symfony\Component\Filesystem\Filesystem.php line 289 {"exception":"[object] (Symfony\\Component\\Filesystem\\Exception\\IOException(code: 0): Cannot rename \"C:\\Programs\\XAMPP\\htdocs\\var\\cache\\prod\\translations\\catA970.tmp\" to \"C:\\Programs\\XAMPP\\htdocs/var/cache/prod\\translations/catalogue..NXhscRe.php\". at C:\\Programs\\XAMPP\\htdocs\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\Filesystem\\Filesystem.php:289)"} []
[2019-06-14 22:40:13] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Filesystem\Exception\IOException: "Cannot rename "C:\Programs\XAMPP\htdocs\var\cache\prod\translations\catAADC.tmp" to "C:\Programs\XAMPP\htdocs/var/cache/prod\translations/catalogue..NXhscRe.php.meta"." at C:\Programs\XAMPP\htdocs\vendor\symfony\symfony\src\Symfony\Component\Filesystem\Filesystem.php line 289 {"exception":"[object] (Symfony\\Component\\Filesystem\\Exception\\IOException(code: 0): Cannot rename \"C:\\Programs\\XAMPP\\htdocs\\var\\cache\\prod\\translations\\catAADC.tmp\" to \"C:\\Programs\\XAMPP\\htdocs/var/cache/prod\\translations/catalogue..NXhscRe.php.meta\". at C:\\Programs\\XAMPP\\htdocs\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\Filesystem\\Filesystem.php:289)"} []
[2019-06-14 22:40:14] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Filesystem\Exception\IOException: "Cannot rename "C:\Programs\XAMPP\htdocs\var\cache\prod\translations\catAD7A.tmp" to "C:\Programs\XAMPP\htdocs/var/cache/prod\translations/catalogue..NXhscRe.php"." at C:\Programs\XAMPP\htdocs\vendor\symfony\symfony\src\Symfony\Component\Filesystem\Filesystem.php line 289 {"exception":"[object] (Symfony\\Component\\Filesystem\\Exception\\IOException(code: 0): Cannot rename \"C:\\Programs\\XAMPP\\htdocs\\var\\cache\\prod\\translations\\catAD7A.tmp\" to \"C:\\Programs\\XAMPP\\htdocs/var/cache/prod\\translations/catalogue..NXhscRe.php\". at C:\\Programs\\XAMPP\\htdocs\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\Filesystem\\Filesystem.php:289)"} []
